src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -157,7 +157,7 @@ public bool IsOptional
         /// </summary>
         internal abstract bool IsMetadataOptional { get; }
 
-        internal abstract bool IsNullChecked { get; }
+        public virtual bool IsNullChecked { get => false; }
```

Comment:
>virtual [](start = 15, length = 7)

Let's leave this as `abstract`, to ensure all implementations are forced to handle this appropriately.